iotdb-core/datanode/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -3015,6 +3026,24 @@ public void setClusterName(String clusterName) {
     this.clusterName = clusterName;
   }
 
+  public synchronized String getClusterId() throws LoadConfigurationException {
+    if (clusterId.isEmpty()) {
+      try (ConfigNodeClient client =
+          ConfigNodeClientManager.getInstance().borrowClient(ConfigNodeInfo.CONFIG_REGION_ID)) {
```

Comment:
get cluster id when register data node in restart . "sendRestartRequestToConfigNode and 
 sendRegisterRequestToConfigNode"

Reply:
This might slow down the startup speed of the DataNode.
When the DataNode is registering, the cluster id may not yet be ready. For compatibility reason, the cluster id will be checked and generated after the state machine is fully recovered, and asynchronously written into the consensus layer. This process can sometimes be slow.

Reply:
I think the ClusterID should be something that a DataNode has when it starts up, and now it seems a bit odd to need to use a try catch every time datanode uses a clusterID

Reply:
now fetch clusterId when dn start and restart